requirements/constraints.txt

```diff
@@ -37,6 +37,9 @@ urllib3>=1.26.5                               # https://github.com/advisories/GH
 # TODO: includes async features https://docs.sqlalchemy.org/en/14/changelog/migration_20.html
 sqlalchemy<2.0
 
+# changed behaviour in using middlewares (https://fastapi.tiangolo.com/release-notes/#0910)
+# TODO: look into failling unit tests and upgrade code
```

Comment:
So we rather do not let TODOs in the code cause from experience they tend to be lost. Rather, you should open an issue in osparc-simcore then write:
```
# NOTE: [link to github issue]
```